biojava-structure-gui/src/test/java/org/biojava/nbio/structure/gui/TestSimilarityCalc.java

```diff
@@ -0,0 +1,98 @@
+package org.biojava.nbio.structure.gui;
```

Comment:
I think this test should go to the integrationtest module. It doesn't seem to be specifically related to structure-gui.